### PR TITLE
Update documentation

### LINKED ISSUE #1148: Update documentation Feb 2022
Documentation and screenshots are out of date and missing info about newer features.

* confidence sliders
* tooltip visibility toggle
* track trails
* lots of the UI has changed.
* lock camera.
* deconflict quick start interpolation and uI guide interpolation pages, combine.
* probably lots more.

Update screenshots, review text, update anything that is outdated.

### PR DESCRIPTION
**Pull request by subdavis**
fixes #1148 

blocked by #1147

### CODE REVIEW

**Comment by BryonLewis** [comment]
> I'd like some early feedback on a couple of concepts. I'm publishing drafts of the new docs here: https://subdavis.com/dive/

I like the bar at the top.  The Support Menu is a little weird because it takes the user further down in the page which immediately hides the top menu.  All the other links reference the top of the page so the menu is still visible.

- The kitware logo on the main screen looks fuzzy (maybe just to me).  Specifically the text below the icon.  This image has a high enough resolution, maybe it's just how it's downscaled or drawn?

These are probably best left for eventual review but I didn't want to forget.:
- In the sharing descriptions there is no reference to tell them to "include subfolders" anymore?
- On export for web we may want to provide a link to what a VIAME.csv is, we provide a link to DIVE Configuration specification.
- Cloning datasets might want to talk about the fact that deleting the source will orphan the clones?

**Comment by marySalvi** [comment]
>  I like the bar at the top. The Support Menu is a little weird because it takes the user further down in the page which immediately hides the top menu. All the other links reference the top of the page so the menu is still visible.

I agree that the support menu is a little weird for the same reason. In addition I think the text next to the Kitware Logo should change to the selected 'Topic', not the first Heading in said topic. 

For example this would be 'User Guide' instead of home:
Screenshot from 2022-02-23 12-52-06

**Comment by subdavis** [comment]
Ready for review, staging site is up to date: https://subdavis.com/dive/

> The Support Menu is a little weird

Support page has been recreated, definitely need some feedback there.

> The kitware logo on the main screen looks fuzzy (maybe just to me)

Rescaled, does this look better?

> In the sharing descriptions there is no reference to tell them to "include subfolders" anymore?

it's still there

> On export for web we may want to provide a link to what a VIAME.csv is, we provide a link to DIVE Configuration specification.

Done

> Cloning datasets might want to talk about the fact that deleting the source will orphan the clones?

Done

> I think the text next to the Kitware Logo should change to the selected 'Topic', not the first Heading in said topic.

This is how mkdocs works, I'm not aware that this is something we can customize.

File: docs/UI-Annotation-View.md [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
 * **Editing Annotation** - Editing annotations are cyan in color and provide handles to resize the annotation as well as a central handle to move the annotation to different spot.
-    * ![Track Edit Mode](images/TrackEditMode.png)
-* **Creating Annotation** - Creating an annotation requires clicking and dragging the mouse.  When the AnnotionView is ready to create the annotation the mouse will turn into a crosshair.
-    * ![Track Creation Crosshair](images/TrackCreationCrosshair.png)
-* **Interpolated Annotation** - If an annotation has interpolation and the current frame isn't a keyframe it will appear slightly faded and will become a keyframe if the user edits the size or position
-    * ![Interpolated Editing](images/InterpolatedEditing.png)
+
+    ![Track Edit Mode](images/TrackEditMode.png)
+
+* **Creating Annotation** - Creating an annotation requires clicking and dragging the mouse.  Creating in the annotation window is indicated by a cursor crosshair.
```
Comment (BryonLewis) [inline docs/UI-Annotation-View.md:23]: We have the type indicator on the mouse cursor as well (rectangle, polygon, line).

File: docs/UI-Attributes.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+1. Click on the ==:material-plus: Attribute== icon for in either the track or detection attribute area.
+    1. ![New Attribute Panel](images/Attributes/newAttribute.png)
+1. Enter a unique name
+1. Choose a Datatype
+    1. `Number`
+    1. `Boolean` (True/False)
+    1. `Text`
+        1. Custom text that the user provides
+        1. A predefined list of text to choose from
```
Comment (BryonLewis) [inline docs/UI-Attributes.md:66]: predefined line separated list?

File: docs/UI-Navigation-Editing-Bar.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+![Navigation Bar Highlighted](images/UIView/NavBarHighlight.png)
+
+## Navigation Bar
+
+The navigation bar is the row of controls at the very top of the window.
+
+* ==:material-database: Data== navigates to the folder that contains the current dataset.
+* ==:material-pipe: Run Pipeline== will launch a pipeline dropdown menu.
+    * **NOTE** Current annotations will be replaced by the pipeline output when it is complete.  You should not perform annotations while a pipeline is running.
```
Comment (BryonLewis) [inline docs/UI-Navigation-Editing-Bar.md:9]: I'm wondering if this note should be referenced in the Pipeline and Training section as well?

File: docs/UI-Track-List.md [hunk 4]
```diff
@@ -2,25 +2,48 @@
...
+
+![Track List Selected Track](images/TrackListSelectedTrack.png){ width=260px align=right }
+
+A track that spans multiple frames and has more options
+
+* ==:material-delete:=={ .error } deletes the entire track
+* ==:material-call-split:== splits the track into 2 smaller tracks on the current frame.
+* ==:material-star:== is filled in if the current frame annotation is a keyfame.  Clicking this will either remove the keyframe if it exists or make the current interpolated annotation a keyframe.
+* ==:material-vector-selection:== turns interpolation on/off for the interval between keyframes.
+* ==:material-chevron-double-left:== jumpts to the first frame of track
```
Comment (BryonLewis) [inline docs/UI-Track-List.md:43]: `jumpts` to `jumps`

File: docs/UI-Type-List.md [hunk 5]
```diff
@@ -2,50 +2,60 @@
...
-**Confidence Filter**  - for pipeline results the confidence filter can be used to filter out different items based on their current confidence level.  As the confidence filter is adjusted the annotations in the AnnotationView as well as the Timeline views will update in realtime.
+Each dataset maintains its own list of types, and types can be defined on-the-fly.
+
+The Type List is used to control visual styles of the different types as well as filter out types that don't need to be displayed.
+
+* The checkbox next to each type name can be used to toggle types on and off.
+* ==:material-sort-alphabetical-ascending:== toggles the sort order between alphabetical and by number of annotations of each type.
+* ==:material-cog:== opens the type settings menu.
+* ==:material-delete:=={ .error } will remove the type from any visible track or delete the track if it is the only type.
+* ==:material-swap-horizontal:== will switch the left pane to the track attribute editor view.
```
Comment (BryonLewis) [inline docs/UI-Type-List.md:17]: is this meant to be 'pane' or is it meant to be 'panel'?

File: docs/Web-Version.md [hunk 6]
```diff
@@ -1,101 +1,63 @@
-# Web Version
-
-[Try our public server ➥](https://viame.kitware.com){ .md-button .md-button--primary }
-
-You can also [run your own](Server-Operator.md).
-
 !!! warning
     **VIAME Web is automatically updated at 2AM EST/EDT every Thursday.**
 
     If you are running a pipeline or training workflow during update, it will be interrupted and started over.
```
Comment (BryonLewis) [inline docs/Web-Version.md:4]: `started over` to `restarted`?  I don't know which one to use.

File: docs/index.md [hunk 7]
```diff
@@ -1,27 +1,63 @@
...
 
-## Definitions
+**Interpolation** - The implicit bounding boxes between keyframes in a track.
 
-* **DIVE** is the annotator and data management software system.  It is our name for the code and capabilities, including both web and desktop, that can be deployed and configured for a variety of needs.
-* **VIAME** (Video and Image Analytics for Marine Environments) is a suite of computer vision tools for object detection, tracking, rapid model generation, and many other types of analysis.  Get more info at [viametoolkit.org](https://www.viametoolkit.org/)
-* **VIAME Web** is the *specific* DIVE Web deployment at [viame.kitware.com](https://viame.kitware.com). It includes a web-based annotator with the capabilities to run VIAME workflows on user-provided data.  You may deploy the web system into your own labratory or cloud environment.
+**Attributes** - Attributes are free-form secondary annotations on both tracks and detections. For example, a `fish` type track may have an `is_adult` boolean attribute.
```
Comment (BryonLewis) [inline docs/index.md:63]: does 'secondary annotations' invoke more visual items than data?

**Review by BryonLewis — CHANGES_REQUESTED** [review]
The logo thing is probably just how I view the scaling the kitware logo on the main page.  I'm sure it's fine.

Thanks for going through and taking all these new screenshots and updating all the documentation.  Most of my changes are comments/questions so feel free to mark them as resolved if you don't feel it is necessary.

File: docs/index.md (hunk 7 above)
Comment (subdavis) [reply docs/index.md]: switched to "secondary characteristic" to complement language in "Types"

File: docs/UI-Navigation-Editing-Bar.md (hunk 3 above)
Comment (subdavis) [reply docs/UI-Navigation-Editing-Bar.md:9]: That kinda feels out of scope for this page, but I will defer to you.

**Review by BryonLewis — APPROVED** [review]
Looks good, thanks for doing this.

**Review by BryonLewis — APPROVED** [review]

**Pull request merged**